### PR TITLE
Update Contributors.md

### PR DESCRIPTION
**Pull request by winniffy**
If you're adding your profile, the title of this PR should be: `[Add profile] -> [Your Name]`

### CODE REVIEW

**Comment by Evavic44** [comment]
This is a duplicate @winniffy. You have already made a PR to add your profile <a href="https://github.com/CommunityPro/Practice-Contribution/pull/47">here</a>.

Kindly close the one you don't want merged

**Comment by winniffy** [comment]
done. @Evavic44

File: Contributors.md [hunk 1]
```diff
@@ -233,4 +233,17 @@
 <!-- End of column-18 -->
 </tr>
 </table>
-
+<!-- Start of column-19 -->
```
Comment (Evavic44) [inline Contributors.md:236]: Kindly add your profile into the table under the `<!-- End of coulmn-18 -->` line and delete the duplicate `</table>` and `</tr>` tags

So change will look like this

```yml
<!-- End of column-18 -->
<!-- Start of column-19 -->
<td align="center">
  <a href="https://github.com/winniffy">
    <img
      src="https://avatars.githubusercontent.com/u/81039726?v=4"
      width="100px"
    />
    <br />
    <sub>Winner Umeh</sub>
  </a>
</td>
<!-- End of column-19 -->
```

**Review by Evavic44 — CHANGES_REQUESTED** [review]

**Comment by winniffy** [comment]
done. kindly check, @Evavic44

File: Contributors.md [hunk 2]
```diff
@@ -231,6 +231,15 @@
...
+<td align="center">
+  <a href="https://github.com/winniffy">
+    <img
+      src="https://avatars.githubusercontent.com/u/81039726?v=4"
+      width="100px"
+    />
+    <br />
+    <sub>Winner Umeh</sub>
+  </a>
+</td>
```
Comment (Evavic44) [inline Contributors.md:244]: Good, now close it with the 

`</tr>` and
`</table>` tags

End result

```
   <td align="center">
    <a href="https://github.com/winniffy">
     <img
      src="https://avatars.githubusercontent.com/u/81039726?v=4"
      width="100px"
    />
    <br />
    <sub>Winner Umeh</sub>
   </a>
   </td>
  </tr>
</table>
```

**Review by Evavic44 — CHANGES_REQUESTED** [review]
Awesome, now close it with the </tr> and </table> tags

**Review by Evavic44 — APPROVED** [review]
Nice work, I'll fix any other issue. 👍🏾

**Pull request merged**